src/edu/washington/escience/myria/daemon/MyriaDriverLauncher.java

```diff
@@ -297,12 +301,17 @@ public static LauncherStatus run(final String[] args)
       final Configuration commandLineConf =
           CommandLine.parseToConfiguration(args, commandLineClasses);
       final Injector commandLineInjector = tang.newInjector(commandLineConf);
-      logStartupMessage(commandLineClasses, commandLineInjector);
+      LOGGER.info(genStartupMessage(commandLineClasses, commandLineInjector));
       final String runtimeClassName = commandLineInjector.getNamedInstance(RuntimeClassName.class);
-      final String driverJobSubmissionDirectory =
-          commandLineInjector.isParameterSet(DriverJobSubmissionDirectory.class)
-              ? commandLineInjector.getNamedInstance(DriverJobSubmissionDirectory.class)
-              : null;
+      final String driverJobSubmissionDirectory;
```

Comment:
Sure, it's a matter of taste whether you prefer consistency or (IMO) readability. I certainly try to use `final` wherever I can.